### ISSUE TITLE
Auto binary failing shipit (plugin is not a constructor)

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#471]
**Describe the bug**

Since v7, `npx` no longer works. To get around that, I've working to convert our code over to use the binary (even going so far as to create a custom orb for it in `auto-it/orbs`). 

Unfortunately the binary is failing too: https://circleci.com/gh/artsy/renovate-config/3409

```
TypeError: plugin is not a constructor
    at Object.loadPlugin [as default] (/snapshot/auto/node_modules/@auto-it/core/dist/utils/load-plugins.js:53:12)
    at /snapshot/auto/node_modules/@auto-it/core/dist/auto.js:1022:67
    at Array.map (<anonymous>)
    at Auto.loadPlugins (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:1022:14)
    at Auto.<anonymous> (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:166:30)
    at step (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:43:23)
    at Object.f [as next] (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:24:53)
    at fulfilled (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:15:58)
    at <anonymous>
Exited with code 1
```

This is the _exactly_ same error as #442, so I'm not sure if it's also a missing dependency issue or if it's a configuration problem between our v6 and v7 versions.

**Comment by zephraph** [comment]
So here's a little extra error info: https://circleci.com/gh/artsy/renovate-config/3410. 

It's failing on the npm plugin... I'm still investigating, I have a suspicion of what may be going on.

**Comment by zephraph** [comment]
Yep, I figured it out! PR in-bound...

**Comment by zephraph** [comment]
This is verifiably fixed thanks to #480
